Fenner_William-DFBT/js/index.js

```diff
@@ -4,10 +4,15 @@
 
 function form_builder(attach, ...form_elements){
```

Comment:
Why not use es6 arrow function? const form_builder = (...) => {}

Reply:
Honestly, I had Webstorm in JSX Harmony mode. I tried a couple different things at the time, but they returned errors on the syntax. But I see what you've done here so I have an idea on what revisions I want to do.